### ISSUE TITLE
13.0.0 - Firebase Credential -The incoming JSON object does not contain a client_email field

### ISSUE THREAD

**Issue by mustafa-barakat** [firebase/firebase-admin-node#2778]
### Describe your environment

  * Operating System version: Docker on Windows
  * Firebase SDK version: 13.0.0
  * Firebase Product: Firebase Admin
  * Node.js version: 18+
  * NPM version: 10.9

#### Steps to reproduce:

- Upgrade to firebase admin to 13.0.0
- When using initializeApp with credential and adding (projectId,clientEmail,privateKey)

This doesn't happen in previous versions

Error
message: 'Credential implementation provided to initializeApp() via the "credential" property failed to fetch a valid Google OAuth2 access token with the following error: "The incoming JSON object does not contain a client_email field"

#### Relevant Code:

```
import { initializeApp } from 'firebase-admin/app';
import { credential } from 'firebase-admin';

initializeApp({
      credential: credential.cert({
        projectId: 'PROJECTID',
        clientEmail:
          'EMAIL,
        privateKey:
          'PRIVKEY',
      }),
    });
```

**Comment by ivstiv** [comment]
I am running into the exact same issue after the release of v13.

- Node 20
- Firebase Admin 13.0.0, 
- Operating System version: node:20-bullseye-slim (debian docker)

This is how I am initialising my app:
```ts
firebaseAdmin.initializeApp({
    credential: firebaseAdmin.credential.cert({
      projectId: "from-env",
      clientEmail: "from-env",
      privateKey: "from-env",
    }),
  },
  "label-here",
);
```

Probably related?: https://github.com/firebase/firebase-admin-node/pull/2466

**Comment by mustafa-barakat** [comment]
> I am running into the exact same issue after the release of v13.
> 
> * Node 20
> * Firebase Admin 13.0.0,
> * Operating System version: node:20-bullseye-slim (debian docker)
> 
> This is how I am initialising my app:
> 
> ```ts
> import firebaseAdmin from "firebase-admin";
> 
> firebaseAdmin.initializeApp({
>     credential: firebaseAdmin.credential.cert({
>       projectId: "from-env",
>       clientEmail: "from-env",
>       privateKey: "from-env",
>     }),
>   },
>   "label-here",
> );
> ```
> 
> Probably related?: #2466

Yes, and for some reason tests were replaced

**Comment by mustafa-barakat** [comment]
Related #2769

**Comment by lahirumaramba** [comment]
Thanks folks, this will be fixed in https://github.com/firebase/firebase-admin-node/pull/2779

**Comment by lahirumaramba** [comment]
This issue is now fixed in `v13.0.1`. Thanks for your patience! Please open a new issue if you run into any problems.

**Comment by Rene0115** [comment]
thanks for this

> This issue is now fixed in `v13.0.1`. Thanks for your patience! Please open a new issue if you run into any problems.
